src/semantic.less

```diff
@@ -10,6 +10,8 @@
   Import this file into your LESS project to use Fomantic-UI without build tools
 */
 
+// stylelint-disable no-invalid-position-at-import-rule
```

Comment:
`& { @import 'definitions/globals/reset'; }` is used to isolate the imported variable scope?

Reply:
Yes, according to  9f750c262f832e979337d43c872f8a2c88c7d3b